CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 # Changelog for the Mapbox Maps SDK for Android
 
 Mapbox welcomes participation and contributions from everyone. Please read [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) to get started.
+## 9.4.0-beta.1 - August 6, 2020
+[Changes](https://github.com/mapbox/mapbox-gl-native-android/compare/android-v9.4.0-alpha.1...android-v9.4.0-beta.1) since [Mapbox Maps SDK for Android v9.4.0-alpha.1](https://github.com/mapbox/mapbox-gl-native-android/releases/tag/android-v9.4.0-alpha.1):
+### Features
+ - Expose OnIndicatorPositionChangedListener which will be invoked when indicator position changes. ([#515](https://github.com/mapbox/mapbox-gl-native-android/pull/515))
```

Comment:
Is this the user location indicator? It would be helpful to be more specific.

Suggestion: Expose `OnIndicatorPositionChangedListener` which is invoked when the position of the user location indicator changes.

Reply:
Thanks for the review, I will address it in a follow-up PR.